app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -60,191 +64,80 @@ keywords:
 
-If you’re accepted on to a teacher training course in England, you’ll train for one year (or longer if you’re part-time) to gain [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts).
 
+## 6. Complete your teacher training and gain QTS
 
-## Apply for your first teaching job
+If you’re accepted on to a teacher training course in England, you’ll train for one year (or longer if you’re part-time) to gain [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
 
-When you have qualified teacher status, you will be able to apply for paid teaching jobs in all schools in England.
 
-To find a teaching job, you must apply to individual schools (not the Department for Education). Use the [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) service to search for jobs by region, subject specialism and age group.
+## 7. Apply for your first teaching job
 
-You may need to apply for a different visa to work in England as a qualified teacher. Our [guidance for non-UK nationals wishing to teach in England](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk) explains the skilled worker visa and covers other ways you can update, extend or switch your visa when you finish your training.
+When you have qualified teacher status, you will be able to apply for paid teaching jobs in all schools in England. Depending on your immigration status, you may also need to apply for a visa allowing you to work in the UK. 
 
-<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#job">Learn more about your first teaching job in England</a>.
+[How to apply for your first teaching job](/non-uk-teachers/find-a-teaching-job-as-a-non-uk-citizen).
 
-## Understand English education
+## Meet non-UK citizens who trained to teach in England
 
-Learn more about:
+$why-train-video$
 
-- [types of English school](https://www.gov.uk/types-of-school)
-- [curriculum and qualifications](https://www.gov.uk/national-curriculum)
-- [behaviour and discipline in schools](https://www.gov.uk/government/publications/behaviour-and-discipline-in-schools)
-- [Office for Standards in Education, Children’s Services and Skills (Ofsted)](https://www.gov.uk/government/organisations/ofsted)
-- [The Education Endowment Foundation](https://educationendowmentfoundation.org.uk/guidance-for-teachers) – a charity gathering international evidence on teaching 5 to 16 year olds
-- [Times Educational Supplement (TES)](https://www.tes.com/) – information and resources for teachers
-- [join the Aspiring Teachers Forum on Facebook](https://www.facebook.com/groups/1357146377672255/)
-- [read interviews with real teachers](/blog)
+[Read the transcript for the 'Why train to teach in England?' video](/non-uk-teachers/why-train-to-teach-in-england-video-transcript).
 
 ## Contact
 
```

Comment:
Again, I would rather remove the link to the TTA page in line 146 - we need people to call us first to check if they are eligible.